libs/application/template-api-modules/src/lib/modules/shared/api/driving-license/sortTeachers.ts

```diff
@@ -0,0 +1,28 @@
+import { Teacher } from '@island.is/clients/driving-license'
+
+const order =
```

Comment:
This something that could be utilized elsewhere in the monorepo as a shared utility?